<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/manage.png</normaloff>../icon/manage.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>1200</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#loginButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255,255,255,210);
	border-radius: 5px;
}
QPushButton#loginButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(105,118,132,200)
}
QPushButton#loginButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}

QPushButton#signupButton:hover{
	font : bold;
}

QCheckBox{
	color: rgba(255,255,255,200)
}
QLabel#label_2
{
	border: 20px
}

QPushButton#forgotButton
{
color: rgba(255,255,255,200);
background-color : rgba(0,0,0,0);
}

QPushButton#forgotButton:hover{
	font : bold;
}





</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1600</width>
       <height>1200</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
width : 100%</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>alo1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>900</width>
       <height>1120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,100);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1600</width>
       <height>200</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255)</string>
     </property>
     <property name="text">
      <string>Login Form</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>240</y>
       <width>700</width>
       <height>65</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 3px solid rgba(255,255,255,255);

color: rgba(255,255,255,230);
padding-bottom: 7px;</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>370</y>
       <width>700</width>
       <height>65</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 3px solid rgba(255,255,255,255);

color: rgba(255,255,255,230);
padding-bottom: 7px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>675</y>
       <width>420</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>550</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Remember me</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>800</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,250)</string>
     </property>
     <property name="text">
      <string>Not a member?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="signupButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>800</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: none;
color: rgba(255,51,51, 220);</string>
     </property>
     <property name="text">
      <string>Signup now</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forgotButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>550</y>
       <width>440</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Forgot your user name or password</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>userName</zorder>
    <zorder>password</zorder>
    <zorder>loginButton</zorder>
    <zorder>checkBox</zorder>
    <zorder>label_6</zorder>
    <zorder>signupButton</zorder>
    <zorder>label_4</zorder>
    <zorder>forgotButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
